[2025-04-13T10:56:38.370+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-13T10:56:38.387+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-13T10:56:36.948170+00:00 [queued]>
[2025-04-13T10:56:38.394+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-13T10:56:36.948170+00:00 [queued]>
[2025-04-13T10:56:38.395+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-13T10:56:38.408+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_raw_data> on 2025-04-13 10:56:36.948170+00:00
[2025-04-13T10:56:38.413+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=330) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-13T10:56:38.414+0000] {standard_task_runner.py:72} INFO - Started process 335 to run task
[2025-04-13T10:56:38.415+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'extract_raw_data', 'manual__2025-04-13T10:56:36.948170+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/lead_data_pipeline.py', '--cfg-path', '/tmp/tmp_cr0b29g']
[2025-04-13T10:56:38.416+0000] {standard_task_runner.py:105} INFO - Job 18: Subtask extract_raw_data
[2025-04-13T10:56:38.478+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-13T10:56:36.948170+00:00 [running]> on host 414b1c8e5c9f
[2025-04-13T10:56:38.554+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-13T10:56:36.948170+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-13T10:56:36.948170+00:00'
[2025-04-13T10:56:38.555+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-13T10:56:38.570+0000] {lead_data_pipeline.py:144} INFO - Started batch processing for 2025-04-13.
[2025-04-13T10:56:38.570+0000] {lead_data_pipeline.py:34} INFO - Created synthetic data for RO country code.
[2025-04-13T10:57:06.335+0000] {lead_data_pipeline.py:101} INFO - Written 100372 records to the CSV file.
[2025-04-13T10:57:06.807+0000] {lead_data_pipeline.py:117} INFO - Added UUID to the dataset.
[2025-04-13T10:57:06.810+0000] {lead_data_pipeline.py:152} INFO - Finished batch processing 2025-04-13.
[2025-04-13T10:57:06.811+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-13T10:57:06.821+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-13T10:57:06.822+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=extract_raw_data, run_id=manual__2025-04-13T10:56:36.948170+00:00, execution_date=20250413T105636, start_date=20250413T105638, end_date=20250413T105706
[2025-04-13T10:57:06.860+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-13T10:57:06.883+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-13T10:57:06.885+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
